src/crypto/tests/BUILD.gn

```diff
@@ -44,6 +43,11 @@ chip_test_suite("tests") {
     "TestCryptoLayer.h",
   ]
 
+  if (chip_device_platform != "efr32") {
```

Comment:
is the heap issue applicable for all variants of EFR32 like MG12, MG21 , MG24 etc ?

Reply:
I only have an MG12 to test